### PR TITLE
chore(tooltip): create codemod for Tooltip component

### PR DESCRIPTION
**Pull request by thebesson**
# Purpose of PR

This PR adds a codemod for Tooltip component from v3 to v4.

## PR Checklist

- [x] I have read the relevant `readme.md` file(s)
- [x] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Storybook stories are added/updated/not required
- [x] Usage notes are added/updated/not required
- [x] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [x] Doesn't contain any sensitive information

### CODE REVIEW

**Review by suevalov — APPROVED** [review]

File: packages/forma-36-codemod/README.md [hunk 1]
```diff
@@ -80,3 +80,6 @@ Migrates `Pill` component from Forma@v3 to Forma@v4
 #### `v4-button`
 
 Migrates `Button` component from Forma@v3 to Forma@v4
+#### `v4-tooltip`
```
Comment (gui-santos) [inline packages/forma-36-codemod/README.md:83]: I think you need to add one line break above this line

that's why the pipeline is failing.... I guess

**Review by gui-santos — APPROVED** [review]

File: packages/forma-36-codemod/README.md (hunk 1 above)
Comment (thebesson) [reply packages/forma-36-codemod/README.md:83]: oh, thanks

File: packages/forma-36-codemod/transforms/v4-tooltip.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+const { modifyPropsCodemod } = require('./common/modify-props-codemod');
+const { pipe } = require('./common/pipe');
+const { getComponentLocalName } = require('../utils');
+const { getFormaImport } = require('../utils/config');
+
+module.exports = modifyPropsCodemod({
```
Comment (massao) [inline packages/forma-36-codemod/transforms/v4-tooltip.js:6]: maybe you don't need this block

File: packages/forma-36-codemod/transforms/v4-tooltip.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    const componentName = getComponentLocalName(j, source, {
+      componentName: 'Tooltip',
+      importName: getFormaImport(),
+    });
+
+    if (!componentName) {
+      return source;
+    }
+    // eslint-disable-next-line no-console
+    console.warn(
```
Comment (massao) [inline packages/forma-36-codemod/transforms/v4-tooltip.js:21]: Maybe add a check for the `content` prop value??
```js
if (hasProperty(modifiedAttributes, { propertyName: 'content' })) {
 typeof  getProperty(modifiedAttributes, { propertyName: 'content' } !== 'string' &&
 // warningMessage... console.warn....
}
```

**Pull request merged**